charon/src/ast/meta.rs

```diff
@@ -113,14 +111,41 @@ pub enum InlineAttr {
     Always,
 }
 
-/// Meta information about an item (function, trait decl, trait impl, type decl, global).
+/// Attributes (`#[...]`).
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    EnumIsA,
+    EnumAsGetters,
+    EnumToGetters,
+    Serialize,
+    Deserialize,
+    Drive,
+    DriveMut,
+)]
+pub enum Attribute {
+    Opaque,
+    Rename(String),
+    Unknown(String),
+}
+
+/// Information about the attributes and visibility of an item, field or variant..
 #[derive(Debug, Clone, Serialize, Deserialize, Drive, DriveMut)]
-pub struct ItemMeta {
-    pub span: Span,
+pub struct AttrInfo {
     /// Attributes (`#[...]`).
     pub attributes: Vec<Attribute>,
     /// Inline hints (on functions only).
     pub inline: Option<InlineAttr>,
+    /// Attribute `charon::rename("...")` provide a custom name that can be used by consumers of
+    /// llbc. E.g. Aeneas uses this to rename definitions in the extracted code.
+    pub rename: Option<String>,
+    /// Whether there was a `#[charon::opaque]` annotation on this item. This is different from
+    /// other reasons this may be opaque.
```

Comment:
I don't understand the second sentence.

Reply:
Should I say "an item may be opaque for other reasons than this attribute, e.g. if specified on the command line"?